### PR TITLE
feat(Core/Player): Added support cast spell for some class spells on login

### PR DESCRIPTION
**Pull request by IntelligentQuantum**
## Changes Proposed:
-  MOVE TO DB
-  

## Issues Addressed:

- Closes 

## SOURCE:

TC

## Tests Performed:

- Tested
- 

## How to Test the Changes:

1. Auto cast these spells on first character login
2. Warrior - Battle Stance
3. Death Knight - Blood Presence

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Nyeriah — APPROVED** [review]

**Comment by Winfidonarleyan** [comment]
- [x] Work

File: data/sql/updates/pending_db_world/rev_1637473250058502604.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1637473250058502604');
+
+DROP TABLE IF EXISTS `playercreateinfo_cast_spell`;
+CREATE TABLE IF NOT EXISTS `playercreateinfo_cast_spell` (
+    `raceMask` INT(10) UNSIGNED NOT NULL DEFAULT '0',
```
Comment (Nyeriah) [inline data/sql/updates/pending_db_world/rev_1637473250058502604.sql:5]: we don't specify length anymore, I can't find where it's documented though @Kitzunu

**Review by Nyeriah — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1637473250058502604.sql (hunk 1 above)
Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1637473250058502604.sql:5]: https://www.azerothcore.org/wiki/sql-standards#tables-and-columns

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1637473250058502604.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    `classMask` INT(10) UNSIGNED NOT NULL DEFAULT '0',
+    `spell` MEDIUMINT(8) UNSIGNED NOT NULL DEFAULT '0',
+    `note` VARCHAR(255) DEFAULT NULL
+) ENGINE=MYISAM DEFAULT CHARSET=utf8;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1637473250058502604.sql:9]: utf8mb4

**Review by Kitzunu — CHANGES_REQUESTED** [review]

**Comment by IntelligentQuantum** [comment]
@Kitzunu Fix it please, i'm bad at sql ^^

**Pull request closed without merging**